src/main/resources/db/migration/V4__alter_user_table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE "member"
+ADD COLUMN email TEXT,
+ADD COLUMN first_name TEXT,
+ADD COLUMN surname TEXT;
```

Comment:
Blank line at end of file
```suggestion
ADD COLUMN surname TEXT;
```